packages/fogbender-vue/src/components/FogbenderFloatingWidget.ts

```diff
@@ -0,0 +1,50 @@
+import { Fogbender } from "fogbender";
+import { defineComponent, inject, onMounted, onUnmounted, ref, watch } from "vue-demi";
+import { fogbender, renderComponent } from "../util";
+export default defineComponent({
+  name: "FogbenderFloatingWidget",
+  props: {
+    verbose: {
+      type: Boolean,
+      required: false,
+    },
+    openInNewTab: {
+      type: Boolean,
+      required: false,
+    },
+  },
```

Comment:
add the rest of floatie props. See `createFloatingWidget`

Reply:
Acknowledged

Reply:
Done